github/enterprise_scim.go

```diff
@@ -193,3 +212,47 @@ func (s *EnterpriseService) ListProvisionedSCIMUsers(ctx context.Context, enterp
 
 	return users, resp, nil
 }
+
+// UpdateAttributeSCIMGroup Update a provisioned group’s individual attributes.
+//
+// GitHub API docs: https://docs.github.com/enterprise-cloud@latest/rest/enterprise-admin/scim#update-an-attribute-for-a-scim-enterprise-group
+//
+//meta:operation PATCH /scim/v2/enterprises/{enterprise}/Groups/{scim_group_id}
+func (s *EnterpriseService) UpdateAttributeSCIMGroup(ctx context.Context, enterprise, scimGroupID string, opts SCIMEnterpriseAttributeOptions) (*SCIMEnterpriseGroupAttributes, *Response, error) {
```

Comment:
`opts` is used for query parameters.

Here we have a body:

<img width="555" height="314" alt="Image" src="https://github.com/user-attachments/assets/0f9fe5de-1950-4a15-bf3f-00c9c0f31dbb" />

We need to rename the fourth parameter:

```suggestion
func (s *EnterpriseService) UpdateAttributeSCIMGroup(ctx context.Context, enterprise, scimGroupID string, attribute SCIMEnterpriseAttribute) (*SCIMEnterpriseGroupAttributes, *Response, error) {
```